incydr/cli/cmds/user_risk_profiles.py

```diff
@@ -0,0 +1,226 @@
+from incydr.cli.cmds.utils import output_models_format
+from incydr.cli.cmds.utils import user_lookup
+from incydr.cli.core import IncydrCommand
+from incydr.cli.core import IncydrGroup
+from incydr.utils import read_dict_from_csv
+
+
+@click.group(cls=IncydrGroup)
+@log_level_option
+@log_file_option
+@click.pass_context
+def user_risk_profiles(ctx, log_level, log_file):
+    """View and manage user risk profiles."""
+    init_client(ctx, log_level, log_file)
+
+
+@user_risk_profiles.command("list", cls=IncydrCommand)
+@table_format_option
+@columns_option
+@profile_filter_options
+@click.pass_context
+def list_(
+    ctx: Context,
+    format_: TableFormat,
+    columns: Optional[str],
+    manager: Optional[str],
+    title: Optional[str],
+    division: Optional[str],
+    department: Optional[str],
+    employment_type: Optional[str],
+    country: Optional[str],
+    region: Optional[str],
+    locality: Optional[str],
+    active: Optional[bool],
+    deleted: Optional[bool],
+    support_user: Optional[bool],
+):
+    """
+    List user risk profiles.
+    """
+    client = ctx.obj()
+    profiles = client.user_risk_profiles.v1.iter_all(
+        manager_id=manager,
+        title=title,
+        division=division,
+        department=department,
+        employment_type=employment_type,
+        country=country,
+        region=region,
+        locality=locality,
+        active=active,
+        deleted=deleted,
+        support_user=support_user,
+    )
+    # TODO with output formatter
+    output_models_format(
+        profiles, "User Risk Profiles", format_, columns, client.settings.use_rich
+    )
+
+
+@user_risk_profiles.command(cls=IncydrCommand)
+@click.argument("user", callback=user_lookup_callback)
+@click.pass_context
+def show(ctx: Context, user):
+    """
+    Show details for a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+    """
+    client = ctx.obj()
+    profile = client.user_risk_profiles.v1.get_user_risk_profile(user)
+    # TODO with output formatter, UserRiskProfile model
+    # tmp:
+    echo(profile.json())
+
+
+@user_risk_profiles.command(cls=IncydrCommand)
+@click.argument("user", callback=user_lookup_callback)
+@click.option(
+    "--start-date",
+    default=None,
+    help="Update a user's starting date. Accepts a date in yyyy-MM-dd (UTC) format.",
+)
+@click.option(
+    "--end-date",
+    default=None,
+    help="Update a user's departure date. Accepts a date in yyyy-MM-dd (UTC) format.",
+)
+@click.option(
+    "--notes", default=None, help="Update the optional notes on a user's profile."
+)
+@click.pass_context
+def update(ctx: Context, user, start_date=None, end_date=None, notes=None):
+    """
+    Update a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+    """
+    # TODO: pass 'CLEAR' to clear field?
```

Comment:
Don't currently have a way to clear fields on update - I was considering having users pass some keyword to the option if they want to clear it.  

A flag that clears all other options passed is also an option but would require some refactoring since then we would want those options to function as flags in the case that a `clear` flag is passed. (ie. something like `update USER --start-date --clear` would clear the start date)

Reply:
How about having both a `--start-date` option and a `--clear-start-date` option, which would need to be incompatible. Then you could easily update one value and clear another in a single call: 

```
incydr user-risk-profile update --clear-end-date --notes "they decided not to quit 🤷"
```